app/presenters/concerns/hyku_addons/collection_presenter_override.rb

```diff
@@ -7,8 +7,8 @@ module CollectionPresenterOverride
     def collection_type
       work_cname = @solr_document.to_h["account_cname_tesim"]
       account_cname = Array.wrap(work_cname).first
-      @all_cname = Account.all.map(&:cname)
-      return super unless @all_cname.include? account_cname
+      @check_cname_exists = Account.exists?(cname: account_cname)
+      return super unless @check_cname_exists.present?
```

Comment:
Why do we need both `.exists?` and `.present?`.  Can't it just be:
````
return super unless Account.exists?(cname: account_cname)
````

Reply:
Thanks I change that